modules/luci-base/luasrc/view/wifi_assoclist.htm

```diff
@@ -58,7 +58,10 @@
 							E('span', wifirate(bss, true)),
 							E('br'),
 							E('span', wifirate(bss, false))
-						])
+						]),
+						'<input class="cbi-button cbi-input-remove" type="button" value="Kick" onclick="kick_client(\'%s\', \'%s\')" />'.format(
+							bss.ifname, 
+							bss.bssid)
 					]);
```

Comment:
Please change this to:

```
E('input', {
    type: 'button',
    class: 'cbi-button cbi-button-remove',
    value: '<%:Disconnect%>',
    'data-bssid': bss.bssid,
    'data-ifname': bss.ifname,
    click: handleDisconnect
})
```

Reply:
![image](https://user-images.githubusercontent.com/14970752/48334152-99855300-e694-11e8-833b-905fd40c6007.png)
![image](https://user-images.githubusercontent.com/14970752/48334210-cf2a3c00-e694-11e8-957c-612fb0eada00.png)

Why not send a HTTP request when click 'disconnect' button using you way ? excuse me.

Reply:
here is two pictures.

Reply:
I guess you're not using the latest master. You need commit https://github.com/openwrt/luci/commit/f4c39dd6ff6f60dc48982d822faa6d9a85c2e4d7 fir this to work correctly.